src/scripts/components/LoginForm.js

```diff
@@ -22,19 +22,19 @@ class LoginForm extends Component<Props, void> {
     return (
       <Form onSubmit={onSubmit}>
         <Input
-          id="login-email"
+          data-test-id="login-email"
```

Comment:
We definitely don't _need_ it, but it is a convention I _think_ could be helpful. Basically by calling it `[data-test-id]` we indicate that this attribute should be used for nothing other than testing. We will be less inclined to do `document.querySelector('#myId')` (instead of doing something in the react way). Not that there is a high chance someone doing that, but it's just a convention I've recently become aware of and kind of like. But feel free to ditch it :)